libraries/chain/chain_controller.cpp

```diff
@@ -300,12 +308,13 @@ transaction_trace chain_controller::_push_transaction(const packed_transaction&
 
          time_point_sec execute_after = head_block_time();
          execute_after += time_point_sec(delay);
-         deferred_transaction dtrx(context.get_next_sender_id(), config::system_account_name, execute_after, trx);
+         //TODO: !!! WHO GETS BILLED TO STORE THE DELAYED TX?
```

Comment:
who gets billed for storing auto-boxed transactions?   the authorizer with the highest delay?

Reply:
bill to highest delay